include/matx/core/tensor_impl.h

```diff
@@ -209,7 +209,7 @@ class tensor_impl_t {
      *   Data type
      */
     template <typename DescriptorType, std::enable_if_t<is_matx_descriptor_v<typename remove_cvref<DescriptorType>::type>, bool> = true>
-    __MATX_INLINE__ tensor_impl_t(T *const ldata,
+    __MATX_INLINE__ __MATX_DEVICE__ __MATX_HOST__ tensor_impl_t(T *const ldata,
```

Comment:
Did you mean to keep the device qualifier here?

Reply:
Yes, this was to work around a compiler bug